src/views/PermissionSend.vue

```diff
@@ -237,7 +254,22 @@ export default {
       } else {
         this.updateSendFees(this.amount)
       }
-    }, 800)
+    }, 800),
+    async feeInUsdValue () {
+      const gas = await this.estimateGas();
```

Comment:
I think it's not good to call this again

Ideally this function should be a computed property and use the component data as much as possible
Just calculating the USD value of whatever the fee is

Reply:
Did it in ideally scenario :)